recipes/firefox/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "68.0.2" %}
+
+package:
+  name: firefox
+  version: {{ version }}
+
+source:
+  url: https://ftp.mozilla.org/pub/firefox/releases/{{ version }}/linux-x86_64/en-US/firefox-{{ version }}.tar.bz2  # [linux64]
+  sha256: 284f58b5ee75daec5eaf8c994fe2c8b14aff6c65331e5deeaed6ba650673357c  # [linux64]
+
+  fn: firefox.pkg  # [osx]
+  url: https://ftp.mozilla.org/pub/firefox/releases/{{ version }}/mac/en-US/Firefox%20{{ version }}.pkg  # [osx]
+  sha256: f92e612fa33f6f944abb83282fd73a024665612332494523e3c3049e0ce04772  # [osx]
+
+  fn: firefox.msi  # [win]
+  url: https://ftp.mozilla.org/pub/firefox/releases/{{ version }}/win64/en-US/Firefox%20Setup%20{{ version }}.msi  # [win]
+  sha256: 090a84963ff15a8881a5f064c530c6e17a559968867ed4fdc3ac1f6d91047bc9  # [win]
+
+build:
+  number: 0
+  binary_relocation:
+    - bin/firefox
```

Comment:
@ocefpaf do we need this?

Reply:
If the prefix changed for you, no. For some reason I needed that to make it work locally. However, I just realized that I'm using an older conda to test.